jinja/base.html

```diff
@@ -63,11 +63,22 @@
     <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/fetch-polyfill@0.8.2/fetch.min.js"></script>
     
     <link href="//cdn-images.mailchimp.com/embedcode/classic-10_7.css" rel="stylesheet" type="text/css">
-<style type="text/css">
-	#mc_embed_signup{background:#fff; clear:left; font:16px Roboto,sans-serif; }
-	/* Add your own Mailchimp form style overrides in your site stylesheet or in this style block.
-	   We recommend moving this block and the preceding CSS link to the HEAD of your HTML file. */
-</style>
+    <style type="text/css">
+      #mc_embed_signup{background:#fff; clear:left; font:16px Roboto,sans-serif; }
+      /* Add your own Mailchimp form style overrides in your site stylesheet or in this style block.
+        We recommend moving this block and the preceding CSS link to the HEAD of your HTML file. */
+    </style>
```

Comment:
I'm curious how come this is marked as changed lines by git? I can't see any difference

Reply:
Just white space indent to align it with the rest of the elements in `<head>`